pallets/nft-manager/src/lib.rs

```diff
@@ -1284,6 +1291,68 @@ impl<T: Config> InnerCallValidator for Pallet<T> {
 
 const STORAGE_VERSION: StorageVersion = StorageVersion::new(4);
 
+pub mod migrations {
```

Comment:
Is the plan to run this migration and then revert the weakbounded change?

Reply:
yes, exactly that.
That is why I have linked this PR with #194 which does exactly that.